### ISSUE TITLE
Try to adjust and extrapolate yearly stats for the current year? Note that current year is partial?

### ISSUE THREAD

**Issue by matkoniecz** [piebro/OpenStreetMap-Statistics#103]
take say https://piebro.github.io/openstreetmap-statistics/#bd16 or edits per year at https://piebro.github.io/openstreetmap-statistics/#fe79

It may look like latest year is a catastrophic collapse - but that is because stats are only from part of the year

Maybe latest year should not be shown? Especially at say edits per year at https://piebro.github.io/openstreetmap-statistics/#fe79

Or maybe it should be extrapolated (so if we have data from just 31 days then multiply by 365/31)? Or noted that last year data are partial?

**Comment by matkoniecz** [comment]
the same may apply for current month in monthly stats

**Comment by piebro** [comment]
Yes, I realized it for some yearly plots last time too. I fixed it for edits per year https://piebro.github.io/openstreetmap-statistics/#fe79 and I think for plots, that would be my preferred way. For tables, I would like to add `2023 (partial)` if necessary because I think the partial data is still interesting.

> the same may apply for current month in monthly stats

I only add full monthly stats by dropping the last month. But if it's, for example, the first of a month, the last changeset snapshot is from last Monday and the stats from a week earlier, the month is only partial. That's why I want to update the stats every second Monday of the month. I could probably document this somewhere or make it more resilient to ensure only full months are added.

**Comment by piebro** [comment]
For now this is kept like it is meaning:
- no partial months
- partial years (adding "(partial)" after the your is possible, but that makes the code overly complex and I hope it's easy to grasp that data from the ongoing year is always partial.)
